code/game/machinery/teleporter.dm

```diff
@@ -217,10 +217,7 @@
 				for(var/rider in L.buckled_mobs)
 					R.force_dismount(rider)
 		//VOREStation Addition End: Prevent taurriding abuse
-		if(prob(5) && !accurate) //oh dear a problem, put em in deep space
-			do_teleport(M, locate(rand((2*TRANSITIONEDGE), world.maxx - (2*TRANSITIONEDGE)), rand((2*TRANSITIONEDGE), world.maxy - (2*TRANSITIONEDGE)), 3), 2)
-		else
-			do_teleport(M, com.locked) //dead-on precision
+		do_teleport(M, com.locked) //dead-on precision
```

Comment:
Add a Chompstation Edit comment per our standards. Otherwise, seems fine, but will leave the decision up to the headmins.

Reply:
Fek, I keep forgetting downstream modular thing.